HowToUseOCR.md

```diff
@@ -24,8 +24,14 @@ Depending on your requirements, you can choose the appropriate language pack for
 
 #### Docker
 
-If you are using Docker, you need to expose the Tesseract tessdata directory as a volume in order to use the additional language packs. 
-#### Docker Compose
+You need to expose the existing Tesseract tessdata directory as a volume in order to use the additional language packs:
+
+    docker run --rm -it --name stirling-pdf-1 frooodle/s-pdf:0.12.2
```

Comment:
These commands are not required right?

Reply:
These commands simply do the extraction for the existing files. The instructions are telling, that they are required to be kept. But when I look at [the init script](https://github.com/Frooodle/Stirling-PDF/blob/main/scripts/init.sh) I'm staring to be doubtful about the whole process. Wouldn't it be enough to make use of the env variable `$TESSERACT_LANGS`? I couldn't find any reference for it in the documentation.

Reply:
TESSERACT_LANGS can be used but that would install them each dock pull instead of just using whats in folder and wouldnt would with anything custom or give the range of light vs heavy lang packs.
TESSERACT_LANGS is only to support digitalOcean mostly.

Those above commands in general are not needed, can you remove them, then ill merge PR

Reply:
What about a check for an existing file before installing it?

    langfile="${LANG}.traineddata"
    [ -f "$langfile" ] || apt-get install -y "tesseract-ocr-$LANG"

Reply:
Seems like a good idea
not to replace manual but to avoid multi download for the var solution

Reply:
> not to replace manual

Why not?

Reply:
I have seen people use custom packs 
Plus there are lite and heavy variants of these Tesseract packs

Reply:
So the docker base copies all existing files in that directory to a temp location 

Then the docker files init.sh script copies them back to that location, the intent being that they are other configs are visible in the users volume and are kept for future updates etc.

If that copy back didn't work I assume it's a read only lock that you put on that volume stopping the copy back from working

Reply:
You're right. The `ro` flag prevented the copy action.

But then this whole PR is useless because it's all about the prevention of this data.